wfl/generate/minimahopping.py

```diff
@@ -2,100 +2,208 @@
 import shutil
 import sys
 import tempfile
+import glob, sys, pathlib
 
 import numpy as np
 from ase.optimize.minimahopping import MinimaHopping
 from ase.io.trajectory import Trajectory
+from ase.io import read, write
 
 from wfl.autoparallelize import autoparallelize, autoparallelize_docstring
 from wfl.utils.misc import atoms_to_list
 from wfl.generate.utils import config_type_append
 from wfl.utils.parallel import construct_calculator_picklesafe
 
 
+def _get_MD_trajectory(rundir):
+	
+	md_traj = []
+	mdtrajfiles = sorted([file for file in glob.glob(f"{rundir}/md*.traj")])

```

Comment:
I have a slight preference for using `pathlib` when possible, so here that'd be `Path(rundir).glob("md*.traj")`.